test/jdk/javax/imageio/plugins/jpeg/JpegWriterWriteNonOpaqueIndexColorModelTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8351108
+ * @summary This test verifies that attempting to write a JPEG using
+ *          ImageIO.write(..) fails by returning `false` for
+ *          translucent IndexColorModels.
+ */
+
+import javax.imageio.ImageIO;
+import java.awt.Transparency;
+import java.awt.image.BufferedImage;
+import java.awt.image.IndexColorModel;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class JpegWriterWriteNonOpaqueIndexColorModelTest {
+    public static void main(String[] args) throws IOException {
+        boolean b1 = testJpegWriter(Transparency.OPAQUE, "OPAQUE", true);
+        boolean b2 = testJpegWriter(Transparency.BITMASK, "BITMASK", false);
+        boolean b3 = testJpegWriter(Transparency.TRANSLUCENT, "TRANSLUCENT", false);
+        if (!(b1 && b2 && b3))
+            throw new Error("Test failed");
+    }
+
+    /**
+     * Test writing a JPEG using a IndexColorModel
+     *
+     * @param imageType a Transparency constant (OPAQUE, BITMASK, TRANSLUCENT) that is used to
+     *                  define a new IndexColorModel.
+     * @param name the name of the Transparency constant
+     * @param expectedWriteReturnValue the expected return value of <code>ImageIO.write(..)</code>
+     * @return true if <code>ImageIO.write(..) == expectedWriteReturnValue</code>, or false otherwise.
```

Comment:
Using `{@code ...}` is the preferred way to mark-up code fragments.

Perhaps, a javadoc is too much… The method isn't that long, and it's rather self-documenting.

Reply:
OK, the javadoc is removed.